src/main/java/redis/clients/jedis/MultiKeyPipelineBase.java

```diff
@@ -8,7 +8,7 @@
 
 public abstract class MultiKeyPipelineBase extends PipelineBase implements
     MultiKeyBinaryRedisPipeline, MultiKeyCommandsPipeline, ClusterPipeline,
-    BinaryScriptingCommandsPipeline, ScriptingCommandsPipeline {
+    BinaryScriptingCommandsPipeline, ScriptingCommandsPipeline, BasicRedisPipeline {
```

Comment:
Oh I see the reason... Is it because methods don't have key to route?
Then I think methods should be treat as multiple key methods. I mean, I think MultiKeyCommandsPipeline is proper place to be placed.